### PR TITLE
mv controller to kubesphere-devops-system

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

File: config/default/manager_auth_proxy_patch.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: controller-manager
+  name: s2ioperator
```
Comment (magicsong) [inline config/default/manager_auth_proxy_patch.yaml]: The name is hard-coded in e2e-test, change the name too in e2e_test.go

File: .gitignore [hunk 2]
```diff
@@ -26,4 +26,3 @@ __*
 *.coverprofile*
 .DS_Store
```
Comment (magicsong) [inline .gitignore]: s2ioperator is generated and changed frequently. Such file should not keep in repo

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -9,8 +9,8 @@ metadata:
 apiVersion: v1
 kind: Service
 metadata:
-  name: controller-manager-service
-  namespace: devops
+  name: s2ioperator
+  namespace: kubesphere-devops-system
```
Comment (magicsong) [inline config/manager/manager.yaml]: namespace is not config in here but in kustomization.yaml

File: .gitignore (hunk 2 above)
Comment (runzexia) [reply .gitignore]: s2ioperator.yaml in the git repository, adding to the gitignore will be easy to misunderstand.
So I deleted the ignore

Comment (magicsong) [reply .gitignore]: you can delete the yaml in this pr

**Pull request merged**